annotation.mustache

```diff
@@ -3,22 +3,23 @@
     <li><h3>{{name}}</h3></li>
     {{#each rows}}
       <li>
-        {{#hasKey this "image"}}
-          <img src="{{{image.url}}}" alt="{{image.text}}" />
+      {{#each this}}
+        {{#hasKey this "_link_array"}}
```

Comment:
I don't guess there's an "is array" check you can do in the template without the `_link_array` hackery?